<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathingPalette</class>
 <widget class="QWidget" name="PathingPalette">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pathing Pallete</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="brushOperationLabel">
     <property name="toolTip">
      <string>The type of operation the brush will apply.</string>
     </property>
     <property name="text">
      <string>Brush Operation Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ToolTypeLayout">
     <item>
      <widget class="QRadioButton" name="replaceType">
       <property name="toolTip">
        <string>Replaces the existing pathing when the brush is used.</string>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">toolTypeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="addType">
       <property name="toolTip">
        <string>Adds to the existing pathing any brush types that are selected.</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">toolTypeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="removeType">
       <property name="toolTip">
        <string>Removes from the existing pathing all the brush types selected.</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">toolTypeGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="brushTypeLabel">
     <property name="toolTip">
      <string>Which pathing type will be applied by the brush.</string>
     </property>
     <property name="text">
      <string>Brush Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="walkable">
       <property name="toolTip">
        <string>Whether the brush will apply unwalkable.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>data/icons/walkable.png</normaloff>Data/Icons/walkable.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushTypeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flyable">
       <property name="toolTip">
        <string>Whether the brush will apply unflyable.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>data/icons/flyable.png</normaloff>Data/Icons/flyable.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushTypeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buildable">
       <property name="toolTip">
        <string>Whether the brush will aply unbuildable</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>data/icons/buildable.png</normaloff>Data/Icons/buildable.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushTypeGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="brushSizeLabel">
     <property name="toolTip">
      <string>The brush size that will be applied on the terrain. The final dimensions are: (size-1)*2+1 x (size-1)*2+1.</string>
     </property>
     <property name="text">
      <string>Brush</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="brushSize1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sets the brush size to 1x1.</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushSizeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushSize3">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sets the brush size to 3x3.</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushSizeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushSize5">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sets the brush size to 5x5.</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushSizeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushSize7">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sets the brush size to 7x7.</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushSizeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushSize9">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sets the brush size to 9x9.</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushSizeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushSize11">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sets the brush size to 11x11.</string>
       </property>
       <property name="text">
        <string>11</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushSizeGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="brushSize">
       <property name="toolTip">
        <string>The current brush size</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>241</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brushSizeSlider">
       <property name="toolTip">
        <string>Sets the brush size.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="brushShapeLayout">
     <item>
      <widget class="QPushButton" name="brushShapeCircle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Data/Icons/Brush/circle.png</normaloff>Data/Icons/Brush/circle.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushShapeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushShapeSquare">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Data/Icons/Brush/square.png</normaloff>Data/Icons/Brush/square.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushShapeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brushShapeDiamond">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Data/Icons/Brush/diamond.png</normaloff>Data/Icons/Brush/diamond.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">brushShapeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>replaceType</tabstop>
  <tabstop>addType</tabstop>
  <tabstop>removeType</tabstop>
  <tabstop>walkable</tabstop>
  <tabstop>flyable</tabstop>
  <tabstop>buildable</tabstop>
  <tabstop>brushSize1</tabstop>
  <tabstop>brushSize3</tabstop>
  <tabstop>brushSize5</tabstop>
  <tabstop>brushSize7</tabstop>
  <tabstop>brushSize9</tabstop>
  <tabstop>brushSize11</tabstop>
  <tabstop>brushSize</tabstop>
  <tabstop>brushSizeSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>brushSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brushSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="brushSizeGroup"/>
  <buttongroup name="brushTypeGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="toolTypeGroup"/>
  <buttongroup name="brushShapeGroup"/>
 </buttongroups>
</ui>
